src/style.css

```
@@ -1 +1,118 @@
-/* add styles here */
+/* #region scale */
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region gray stars */
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+/* #endregion */
+
+/* #region stars--1 */
+.stars--1 .stars__star:nth-child(-n + 1) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.... {
```
You can use commas for grouping selectors and avoid repeating code